.github/workflows/push_main.yml

```diff
@@ -0,0 +1,41 @@
+name: Create Sample Release
```

Comment:
Same here please explain what this is doing in the readme.

Is this something that happens locally or during deplyment?

Reply:
Explain about the github token